bug: un-finished loose end of functions/user

```
FAIL tests/entity/user.test.ts
  ● Test suite failed to run
    tests/entity/user.test.ts:3:22 - error TS2307: Cannot find module '../../src/functions/user' or its corresponding type declarations.
    3 import { save } from '../../src/functions/user'
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/packages/database/src/api/module/get/codes.ts
+++ b/packages/database/src/api/module/get/codes.ts
@@ -1,4 +1,0 @@
-import { analyze } from '../../analyze'
-import { Module } from '../../../entity/Module'
-
-analyze(Module.getCodes)

--- a/packages/database/src/api/module/get/index.ts
+++ b/packages/database/src/api/module/get/index.ts
@@ -1,4 +1,0 @@
-import { analyze } from '../../analyze'
-import { Module } from '../../../entity/Module'
-
-analyze(Module.get)

--- a/packages/database/src/api/module/pull.ts
+++ b/packages/database/src/api/module/pull.ts
@@ -1,4 +1,0 @@
-import { Module } from '../../entity/Module'
-import { analyze } from '../analyze'
-
-analyze(Module.pull)

--- a/packages/database/src/api/moduleCondensed/fetch.ts
+++ b/packages/database/src/api/moduleCondensed/fetch.ts
@@ -1,4 +1,0 @@
-import { analyze } from '../analyze'
-import { ModuleCondensed } from '../../entity/ModuleCondensed'
-
-analyze(ModuleCondensed.fetch)

--- a/packages/database/src/api/moduleCondensed/get.ts
+++ b/packages/database/src/api/moduleCondensed/get.ts
@@ -1,4 +1,0 @@
-import { analyze } from '../analyze'
-import { ModuleCondensed } from '../../entity/ModuleCondensed'
-
-analyze(ModuleCondensed.get)

--- a/packages/database/src/api/user/canTakeModule.ts
+++ b/packages/database/src/api/user/canTakeModule.ts
@@ -1,4 +1,0 @@
-import { canTakeModule } from '../../functions/user'
-import { analyze } from '../analyze'
-
-analyze(canTakeModule)

--- a/packages/database/src/api/user/save.ts
+++ b/packages/database/src/api/user/save.ts
@@ -1,4 +1,0 @@
-import { save } from '../../functions/user'
-import { analyze } from '../analyze'
-
-analyze(save)

--- a/packages/database/tests/entity/user.test.ts
+++ b/packages/database/tests/entity/user.test.ts
@@ -1,8 +1,8 @@
 import { setup } from '../setup'
 import { endpoint } from '../../src/data-source'
-import { save } from '../../src/functions/user'
 import { container, AppDataSource } from '../../src/data-source'
 import { User } from '../../src/entity'
+import { UserProps } from '../../types/modtree'
 
 beforeAll(async () => {
   await setup()
@@ -11,8 +11,16 @@ beforeAll(async () => {
 jest.setTimeout(20000)
 
 test('canTakeModule is successful', async () => {
-  // currently adds Khang
-  await save()
+  const props: UserProps = {
+    displayName: 'Nguyen Vu Khang',
+    username: 'nvkhang',
+    modulesCompleted: ['MA2001'],
+    modulesDoing: ['MA2219'],
+    matriculationYear: 2021,
+    graduationYear: 2025,
+    graduationSemester: 2,
+  }
+  await User.save(props)
 
   const res = await endpoint(
     async () =>

